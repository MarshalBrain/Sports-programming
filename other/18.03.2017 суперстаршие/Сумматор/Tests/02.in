1 100
2 58 84
3 93 6055
2 35 42
3 56 4961
2 54 70
3 56 2112
3 65 -5904
3 33 -7052
3 21 -4546
2 2 18
3 44 -1412
3 36 -2705
2 50 85
3 76 720
3 95 -9036
3 24 -2081
3 10 6388
3 96 -9591
3 30 -9071
2 5 84
2 77 84
2 24 73
2 47 80
3 70 6663
2 32 47
3 71 4976
2 14 91
2 37 87
2 51 82
3 40 -1272
2 63 69
2 39 48
2 1 81
2 52 81
3 88 -3784
2 72 93
2 2 5
2 48 64
3 79 -7039
3 1 7051
2 24 75
2 58 67
3 17 -760
2 48 52
2 40 81
2 29 36
2 58 80
3 53 7607
2 13 50
2 72 91
2 32 67
2 24 79
3 18 5472
3 47 -1046
3 8 -4415
3 22 5319
3 58 -81
3 93 -1886
3 49 5999
3 68 2455
3 26 -685
3 71 -5243
3 70 -5297
3 23 -5327
3 91 -2591
3 14 -8641
3 55 1912
3 41 -5679
3 34 -7800
3 83 -5220
3 34 -2325
3 96 -8300
3 94 -6753
3 49 5044
3 8 -893
3 47 -5634
3 73 -3321
3 56 1435
3 79 7969
3 83 -2078
3 3 1938
3 49 -1354
3 58 -3775
3 67 -4019
3 68 -2684
3 51 -7480
3 56 1447
3 85 8000
3 51 283
3 15 5765
3 86 1902
3 6 8148
3 60 2264
3 11 8681
3 36 -4545
3 65 -8328
3 98 -4697
3 83 -8967
3 84 2614
3 74 -8802
