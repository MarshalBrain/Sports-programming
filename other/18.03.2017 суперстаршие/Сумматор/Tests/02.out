-84
-42
-112
-196
0
0
-60
-60
-299
-299
-18
-18
-299
-18
-536
-583
-442
-1206
-1368
0
-279
-1212
-329
-964
-188
-329
-1478
-2046
-1160
-1778
-556
-1778
-1778
-329
-2046
-238
-1478
-964
-706
-2046
-706
-2046
-2046
-1160
-188
-964
-1962
-1478
-2046
-2046
-104
-1160
-1478
-1778
-1778
-1245
-1478
-2046
-1245
-329
-2046
-188
-1709
-188
-748
-1778
-2046
-2046
-2046
-1962
